bundle/definition/validators.go

```diff
@@ -0,0 +1,29 @@
+package definition
+
+import (
+	"github.com/qri-io/jsonschema"
+)
+
+// ContentEncoding represents a "custom" Schema property
+type ContentEncoding string
```

Comment:
As `NewContentEncoding` is returning a `jsonschema.Validator`, do we really need to expose that type?

Reply:
I don't have strong opinion on it, I just have the habit of reducing the exposed parts to their minimum 👍